docs/examples/src/talk_to_a_document.py

```diff
@@ -0,0 +1,42 @@
+import base64
+
+import requests
+from attrs import define
+
+from griptape.artifacts import GenericArtifact, TextArtifact
+from griptape.drivers import AnthropicPromptDriver
+from griptape.structures import Agent
+
+
+@define
+class DocumentArtifact(GenericArtifact):
+    """Artifact for storing a document.
+
+    Subclassing `GenericArtifact` to avoid printing out `self.value` as `GenericArtifact` does.
```

Comment:
Feels a little weird to me that this is subclassing is needed for this reason only. Doesn't this mean that everyone using the feature will end up creating this subclass?

Reply:
Yeah I don't love it. Let me see if I can provide a more elegant solution to log truncating.